cmdeploy/src/cmdeploy/cmdeploy.py

```diff
@@ -33,10 +34,33 @@ def init_cmd_options(parser):
 def init_cmd(args, out):
     """Initialize chatmail config file."""
     if args.inipath.exists():
-        out.red(f"Path exists, not modifying: {args.inipath}")
-        raise SystemExit(1)
-    write_initial_config(args.inipath, args.chatmail_domain)
-    out.green(f"created config file for {args.chatmail_domain} in {args.inipath}")
+        print(f"Path exists, not modifying: {args.inipath}")
+    else:
+        write_initial_config(args.inipath, args.chatmail_domain)
+        out.green(f"created config file for {args.chatmail_domain} in {args.inipath}")
+    dns = DNS()
+    ipaddress = dns.resolve(args.chatmail_domain)
+    mta_ipadress = dns.resolve("mta-sts." + args.chatmail_domain)
+    entries = 0
+    to_print = ["Now you should add %dnsentry% at your DNS provider:\n"]
+    if not ipaddress:
+        entries += 1
+        to_print.append(f"\tA\t{args.chatmail_domain}.\t\t<your server's IPv4 address>")
+    if not mta_ipadress or mta_ipadress != ipaddress:
+        entries += 1
+        to_print.append(
+            f"\tCNAME\tmta-sts.{args.chatmail_domain}.\t{args.chatmail_domain}."
+        )
+    if entries == 1:
+        singular = "this entry"
+    elif entries == 2:
+        singular = "these entries"
+    else:
+        return
+    to_print[0] = to_print[0].replace("%dnsentry%", singular)
+    for line in to_print:
+        print(line)
```

Comment:
why not print out both lines unconditionally and avoid the singular/plural shenenigans? 
Maybe in green if it resolves correctly, red if not-exist/wrongly?